src/bgnbd_CET.cpp

```diff
@@ -0,0 +1,110 @@
+#include <RcppArmadillo.h>
+#include <math.h>
+#include "clv_vectorized.h"
+
+//' @name bgnbd_CET
+//' @title BG/NBD: Conditional Expected Transactions without covariates
+//'
+//' @description
+//' Calculates the expected number of transactions in a given time period based
+//' on a customer's past transaction behavior and the BG/NBD model parameters.
+//'
+//' @template template_params_bgnbd
+//' @param nPeriods time prediction time frame
+//' @template template_params_rcppxtxtcal
+//' @template template_params_rcppcovmatrix
+//' @template template_params_rcppvcovparams
+//'
+//' @template template_details_paramsbgnbd
+//'
+//' @templateVar name_params_cov_life vCovParams_life
+//' @templateVar name_params_cov_trans vCovParams_trans
+//' @template template_details_rcppcovmatrix
+//'
+//' @return
+//' Returns a vector with the conditional expected transactions for the existing
+//' customers in the BG/NBD model.
+//'
+//' @template template_references_bgnbd
+//'
+arma::vec bgnbd_CET(const double r,
+                    const arma::vec& vAlpha_i,
+                    const arma::vec& vA_i,
+                    const arma::vec& vB_i,
+                    const double nPeriods,
```

Comment:
im not very fond of this name (nPeriods). The n suggests integer in hungarian notation which is not the case. maybe dPeriods. or any other suggestion? the name in pnbd is not better (dPrediction_period).